svcdef/svcdef.go

```diff
@@ -0,0 +1,347 @@
+package svcdef
+
+// consider renaming to census, pbregistry, or just to "essence"
```

Comment:
This comment was intended as a note for myself on potential names for this package, and needs to be removed.